common/src/test/java/com/sequenceiq/cloudbreak/telemetry/fluent/cloud/AdlsGen2ConfigGeneratorTest.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.cloudbreak.telemetry.fluent.cloud;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.cloudbreak.common.exception.CloudbreakServiceException;
+
+public class AdlsGen2ConfigGeneratorTest {
+
+    private final AdlsGen2ConfigGenerator underTest = new AdlsGen2ConfigGenerator();
+
+    @Test
+    public void testGenerateStorageConfigWhenNoAccountIdInThePath() {
+        String loc = "abfs://logs@.dfs.core.windows.net";
```

Comment:
thanks to put this here, many of these tests are covered in FluentConfigServiceTest but those needs to be moved here in the future